ui/push-health/JobListMetric.jsx

```diff
@@ -19,14 +20,15 @@ export default class JobListMetric extends React.PureComponent {
         <div>
           {details.length ? (
             details.map(job => (
-              <Job
-                job={job}
-                jobName={job.job_type_name}
-                jobSymbol={job.job_type_symbol}
-                repo={repo}
-                revision={revision}
-                key={job.id}
-              />
+              <Row key={job.id} className="mt-2">
```

Comment:
I decided to make these lists vertical, because each job can show a couple pieces of information about it (classified or pre-existing in the parent) and having them on a horizontal line was very busy and confusing looking.  Also, since you can easily collapse these sections, it makes the vertical space usage not so critical.